tests/cc56/input.dat

```diff
@@ -39,9 +39,9 @@ compare_values(ref0A2, core.variable("CCSD ROOT 0 (IN A2) TOTAL ENERGY"), 6, "Ro
 ### Dipole check
 
 test_data = {                   # TEST
-(0, "B1", 0, "A1"):  1.0109909, # TEST
-(1, "B1", 1, "A1"):  0.4953101, # TEST
-(0, "A2", 2, "B2"): -0.0535458, # TEST
+(0, "B1", 0, "A1"):  0.9924065, # TEST
+(1, "B1", 1, "A1"): -0.1336723, # TEST
```

Comment:
this second one is a jump -- check through cfour sometime?

Reply:
Sounds good. I don't think Emory has CFOUR, but if we do, I'll be happy to run this before the PR is merged on. I'll check on Monday.

Reply:
I can make cfour generate all three new transition dipoles, so I think this is good to go whenever Jonathon wants to merge.

Orbital ordering whyfors at psicon sounds interesting :-)